### PR TITLE
Updated the scripts for dev and prod, removed ./dist folder and added…

### PR DESCRIPTION
**Pull request by nlukic97**
… it to .gitignore. Updated README.

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -0,0 +1,7 @@
+PORT=
+
+DB_NAME=
+DB_HOST=
+DB_USER=
+DB_PASS=
+DB_DIALECT=
```
Comment (vuxor) [inline .env.example:7]: Add some default values so we can have some detailed explanation about variables

File: ts/app.ts [hunk 2]
```diff
@@ -1,25 +1,124 @@
...
+  .authenticate()
+  .then(() => {
+    console.log('Connection has been established successfully.')
+  })
+  .catch(error => {
+    console.error('Unable to connect to the database:', error)
+  })
+
+// User Model
+const User = sequelize.define(
```
Comment (vuxor) [inline ts/app.ts:35]: Move user model to separate file

File: ts/data.ts [hunk 3]
```diff
@@ -1,10 +1,8 @@
-interface ResponseData {
-    status: number;
-    message: string;
-  }
-
-function createResponseData(responseData: ResponseData){
```
Comment (vuxor) [inline ts/data.ts:6]: Remove also every reference to this function in other files

**Review by vuxor — CHANGES_REQUESTED** [review]

File: .env.example (hunk 1 above)
Comment (nlukic97) [reply .env.example]: I have added default values to the [.env.example file](https://github.com/nlukic97/uteam-api/commit/e2f5d29381d9842633e180bc8d3587b924e4a9d6#diff-a3046da0d15a27e89f2afe639b25748a7ad4d9290af3e7b1b6c1a5533c8f0a8c)

File: ts/app.ts (hunk 2 above)
Comment (nlukic97) [reply ts/app.ts]: Created a [model for the user](https://github.com/nlukic97/uteam-api/commit/e2f5d29381d9842633e180bc8d3587b924e4a9d6#diff-8c323df1da2671b1abf06a669bb39a9deacf54a0ed5bf4b83f6079c28227720b)

File: ts/data.ts (hunk 3 above)
Comment (nlukic97) [reply ts/data.ts:6]: Removed all imports for this module, and commented it: https://github.com/nlukic97/uteam-api/pull/1/commits/9384be18ee04774b918d1adfdbeb91a1066e84a2#diff-1d9cad3d3df20018a61120bb08d5f5ae70fa6e9a78250eb02d30a3605d009614

**Comment by nlukic97** [comment]
Hey @vuxor, I have made the changes you've requested (commented in the code conversations above). 

I have also:
- Structured the app to have an MVC like architecture, by creating a controller [for the user](https://github.com/nlukic97/uteam-api/pull/1/files#diff-78cee36d0b36a1c2bbab287cd94af0fb80fdf7568713df0b1e7c9600ea350303) and a general [App controller](https://github.com/nlukic97/uteam-api/pull/1/files#diff-0b5a5b5580364f2e66708c5cb2a5eb3a653f7e4c504184f28cf0cd84944c57a2) (for the wildcard '*' route)
- Created a [Seeder](https://github.com/nlukic97/uteam-api/pull/1/files#diff-c34ea038f0feff1f9a951391ccec8ef5f34c2a8bbea9c819bfe7eb2a8a5add79), currently for the User model
- Created a way to generate a Model, a Seeder (or both at the same time) from the command line by running npm scripts (similar to Laravel): https://github.com/nlukic97/uteam-api/pull/1/files#diff-1736ddf49a6452d01dbbf6f50f828641c7287cdb87fa48035419745794f217b8 . More about that in the [ts/utils/README.md](https://github.com/nlukic97/uteam-api/tree/dev/ts/seeders) file.

File: ts/app.ts (hunk 2 above)
Comment (nlukic97) [reply ts/app.ts]: https://github.com/nlukic97/uteam-api/blob/e2f5d29381d9842633e180bc8d3587b924e4a9d6/ts/models/User.ts#L1-L28

**Pull request merged**